src/main/java/com/nokia/uwr/controller/callscontroller/CallsController.java

```diff
@@ -0,0 +1,40 @@
+package com.nokia.uwr.controller.callscontroller;
+
+import com.nokia.uwr.model.Measurements;
+import com.nokia.uwr.service.callshandler.CallsHandler;
+import lombok.RequiredArgsConstructor;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+/**
+ * Controller for calls.
+ *
+ * @author MiSobecki
+ */
+@RestController
+@RequestMapping("/api/calls")
```

Comment:
Why there is created other endpoint than endpoint in network-simulator?
https://github.com/PiotrStoklosa/network-simulator/blob/master/src/main/java/com/nokia/uwr/service/rest/RESTAPIClient.java

Reply:
Okay, I will change this in simulator in the next story